config/config.go

```diff
@@ -282,6 +284,9 @@ func saveConfig(cfg *Config) *Config {
 			core.AutoComplete = true
 			core.Output = JSON
 		}
+		if !conf.Section(ini.DEFAULT_SECTION).HasKey("postrequest") {
+			core.PostRequest = true
```

Comment:
seems default PostRequest is true, so should set to false here when "postrequest" key not present?

Reply:
Good point @sureshanaparti. @kevin-lii can you please check this?